publish_file_message.go

```diff
@@ -177,8 +203,22 @@ func (o *publishFileMessageOpts) validate() error {
 	return nil
 }
 
+// buildRawTextMessage creates the appropriate message structure for raw text mode
+func (o *publishFileMessageOpts) buildRawTextMessage() interface{} {
```

Comment:
Inconsistent Message Structure Handling
  publish_file_message.go:207-218
  Issue: The buildRawTextMessage() function only handles PNPublishFileMessage type but not PNPublishFileMessageRaw. However, in validation at line 186-198, you handle PNPublishFileMessageRaw and set o.UseRawText = true, but
   then buildRawTextMessage() won’t work correctly for that case.
  Problem: If someone passes a PNPublishFileMessageRaw directly, the buildRawTextMessage() will return o.MessageText (which might be empty), not the actual message from the PNPublishFileMessageRaw struct.
  Solution: Handle both types in buildRawTextMessage():